README.md

```diff
@@ -52,17 +46,9 @@ Visit the **feedback** section in our [forums](https://forums.improbable.io) and
 **Github issues**<br/>
 Create an issue [in this repository](https://github.com/spatialos/UnityGDK/issues).
 
-## Support policy
-
-We are not supporting the alpha release for game development, as this version is for evaluation only. However, please do give us your feedback.
-
-## Migration from the SpatialOS Unity SDK
-
-Currently the GDK is in its early development, so there is no migration path from the [SpatialOS SDK for Unity](https://github.com/spatialos/UnitySDK) to the GDK. There will be a migration path in later releases.
-
 ---
-* Version: alpha (for evaluation only)
-* Unity Game Development Kit repository: [github.com/spatialos/gdk-for-unity](https://github.com/spatialos/UnityGDK)
+* Version: alpha 
+* The SpatialOS GDK for Unity repository: [github.com/spatialos/gdk-for-unity](https://github.com/spatialos/UnityGDK)
```

Comment:
at the very end the license link: should it point to the license in the repo or in the spatialos documentation.

if in the repo: we need to change the name from Unity GDK to SpatialOS GDK for Unity

Reply:
There's two license links:
(1)  ours (linked to_
(2) Unity's (text requested by lega) - I couldn't find a sensible, linkable one to link to for Unity so have left it - if you know where one is, please LMK as it would be cool.

Will amend other text.